crates/e2e/tests/e2e/setup/mod.rs

```diff
@@ -95,6 +95,7 @@ where
     let http = create_test_transport(NODE_HOST);
     let web3 = Web3::new(http);
     let resetter = Resetter::new(&web3).await;
+    services::clear_database().await;
```

Comment:
Also clearing before the test as I noticed during local tests that the DB doesn't get cleared when you abort the test. That caused this particular e2e to then fail indefinitely because the indexing logic couldn't find the left over tx hashes.